ddl/verify/datecast.sql

```diff
@@ -0,0 +1,11 @@
+-- Verify ccdb:datecast on pg
+
+BEGIN;
+
+SELECT 1/COUNT(*) 
```

Comment:
sqitch needs to throw an sql exception in order to say a validation query has failed. In a handful of the sqitch examples I found, a divide by 0 exception seems like the most popular way.

If the query returns anything, it will pass and the migration will be validated.
If the query doesn't return anything, a  divide by 0 happens.